PatientWebApplication/HealthClinic.DAL/Repository/PatientsRepository.cs

```diff
@@ -1,14 +1,18 @@
 ﻿using HealthClinic.CL.DbContextModel;
 using HealthClinic.CL.Model.Patient;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace HealthClinic.CL.Repository
 {
-    public class PatientsRepository 
+    public class PatientsRepository : IPatientsRepository
     {
         private readonly MyDbContext dbContext;
-        public PatientsRepository(MyDbContext dbContext)
+        public PatientsRepository()
         {
-            this.dbContext = dbContext;
+            var options = new DbContextOptionsBuilder<MyDbContext>().UseMySql("Server=localhost;port=3306;Database=MYSQLHealtcareDB;user=root;password=root").UseLazyLoadingProxies().Options;
+            this.dbContext = new MyDbContext(options);
```

Comment:
The 'DbContextOptionsBuilder' object does not need to be instantiated, it can be passed as a 'MyDbContext' constructor parameter.